src/database/queries/getData.js

```diff
@@ -0,0 +1,15 @@
+const connect = require('./../config/db_connection');
+
+const checkPassword = (pass) => {
```

Comment:
@Angham116 I think this function would be more useful if it always checked for both a name and password (so that you can check if a user's password is correct, for their name, when they attempt to log in).
There could be many users that all have the same password (like `1234` or `P@ssw0rd` 😄) so this function, as it's written right now, would return multiple users if there were multiple users with that same password. I think it would benefit from having `name` (or `email`, if that's what your database uses as the user identifier) as an additional argument so that you can use it to validate a login attempt.